src/map/packets_struct.h

```diff
@@ -5817,6 +5817,18 @@ struct PACKET_ZC_SPECIALPOPUP {
 DEFINE_PACKET_HEADER(ZC_SPECIALPOPUP, 0x0bbe);
 #endif  // PACKETVER_MAIN_NUM >= 20221005
 
+#if PACKETVER >= 20140430
```

Comment:
packet should start from 20140611

Reply:
I kept this one for 20140430 as it is CZ packet